### DISCUSSION TITLE
Mobile compatible?

### DISCUSSION THREAD

**Discussion by spasticginger** [mProjectsCode/obsidian-meta-bind-plugin#53] [General]
I'm loving the meta-bind plugin while using PC, but I've noticed it doesn't seem to be working correctly on mobile.  I've tried on both iPad & an Android phone, but the toggles, sliders, etc. that work fine on PC function but don't update the metadata.  If the note or app is reloaded the position/changes are lost. 

Is there possibly something I don't have enabled, or is this a 'PC-only' plugin?

**Comment by mProjectsCode** [comment]
Version `0.4.0` of the plugin requires Obsidian version `1.1.0` or higher. This is because of obsidians new metadata API. 
Obsidian version `1.1.0` is not yet available on mobile. Because of that, version `0.4.0` of the plugin should not yet be installable on mobile. 
If you are facing this issue you are probably syncing your vault from desktop to mobile.

As a temp fix, you can install a version before 0.4.0.

**Comment by AndriiPetrovDev** [comment]
I'm using the last version 1.4.1(83), API version 1.0.3. And the plugin doesn't work on iPhone 13 iOS 16.3

**Comment by ribber** [comment]
I'm using the last version 1.4.3(95), API version 1.1.16. And the plugin doesn't work on Android 12

**Reply by mProjectsCode** [reply]
Then please open a bug report describing the issue in detail.
